File: Counting/3442.Maximum Difference Between Even and Odd Frequency I/Solution.java
class Solution {
    public int maxDifference(String s) {
        int[] cnt = new int[26];
        for (char c : s.toCharArray()) {
            cnt[c - 'a']++;
        }
        int mi = 110, mx = 1;
        for (int v : cnt) {
            if (v == 0) {
                continue;
            }
            if (v % 2 == 1) {
                mx = Math.max(mx, v);
            } else {
                mi = Math.min(mi, v);
            }
        }
        return mx - mi;
    }
}

File: Counting/3442.Maximum Difference Between Even and Odd Frequency I/Solution.py
class Solution:
    def maxDifference(self, s: str) -> int:
        cnt = [0] * 26
        for c in s:
            x = ord(c) - ord('a')
            cnt[x] += 1
        mi, mx = 110, 1
        for v in cnt:
            if not v:
                continue
            if v % 2:
                mx = max(mx, v)
            else:
                mi = min(mi, v)
        return mx - mi


